### DISCUSSION TITLE
depot is not avaliable for this account

### DISCUSSION THREAD

**Discussion by pepe2095** [SteamRE/DepotDownloader#414] [Q&A]
so i tried to downgrade dlc to game and play it but depotdownloader says:
Connecting to Steam3... Done!
Logging anonymously into Steam3... Done!
Using Steam3 suggested CellID: 183
Got AppInfo for 346110
Using app branch: 'Public'.
Depot 708770 (ARK: Aberration - Expansion Pack (708770) Depot) is not available from this account.
Total downloaded: 0 bytes (0 bytes uncompressed) from 0 depots
Disconnected from Steam

**Comment by SlejmUr** [comment]
`Logging anonymously into Steam3... Done!`
That's is your problem, anonymous account not own dlc or the game itself.
Try add `-username {steamloginname} -remember-password` after your downloading parameters
